en/sensor/airspeed.md

```diff
@@ -39,7 +38,21 @@ Enable each type using its [corresponding parameter](../advanced_config/paramete
 - **TE MS5525:** [SENS_EN_MS5525DS](../advanced_config/parameter_reference.md#SENS_EN_MS5525DS)
 - **Eagle Tree airspeed sensor:** [SENS_EN_ETSASPD](../advanced_config/parameter_reference.md#SENS_EN_ETSASPD)
 
-### Sensor-specific configuration
+You should also check [ASPD_PRIMARY](../advanced_config/parameter_reference.md#ASPD_PRIMARY) is `1` (the default).
+
+### Multiple Airspeed Sensors
```

Comment:
@sfuhrer I'm going to merge this, but would be good if you could check it, and we might want to adjust.

I assume:
1. The airspeed selection is continuous - i.e. it is run to check the airspeed sensor is valid all the time, and triggers the fallback automatically, independent of EFK.
2. Can EKF decide it does not trust the incoming data and force a new selection? Or does it just reset on some check?
3. What checks does the airspeed selector do? If we know what they are I can perhaps find a place here for that.
4. If the EKF does further checks to validate, what are they?

Reply:
1. correct.
2. no there is no EKF->airspeed selector communication. The EKF gets from the selector what the selector thinks is best, and then can decide if it wants to consume the airspeed data or not.
3. You can see a rough explanation on the checks within the parameters. See ASPD_DO_CHECKS. If you need more info let me know, I also would have some example plots to add (motivation for the checks).
4. EKF fuses airspeed if it's above EKF2_ARSP_THR and has a reasonably low innovation compared to groundspeed minus windspeed.

Reply:
Thank you! Captured in https://github.com/PX4/PX4-user_guide/pull/2049